Getting Started.md

```diff
@@ -160,31 +171,27 @@ When Elm shows the type of a functions, it uses arrows `->`.  The rightmost type
 "Hello, Janice" : String
 ```
 
-### Step ...
+### Step 9
 
-Exit the Elm REPL by typing `:exit`, or `CTRL-C` (`CMD-C` on Mac).
+Exit the Elm REPL by typing `:exit`, or `CTRL-C` (`CMD-D` on Mac).
```

Comment:
![screen shot 2016-10-13 at 9 33 17 am](https://cloud.githubusercontent.com/assets/8811312/19357831/1c3692d8-9128-11e6-8871-ae726b479d68.png)

CMD is definitely wrong, but you don't get this error when you exit with CTRL-C?

It exits cleanly for me with CTRL-D, but not CTRL-C.